ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/EthSignerBaseCommand.java

```diff
@@ -51,6 +51,13 @@
     footer = "EthSigner is licensed under the Apache License 2.0")
 public class EthSignerBaseCommand implements Config {
 
+  @Option(
+      names = "--downstream-http-tls-known-servers-file",
+      description = "Path to a file containing the fingerprints of authorized servers",
+      arity = "1",
+      required = false)
```

Comment:
nit: question is going to become, should we authenicate servers against whitelist, or CA, and do we need cmdline args to differentiate?

Reply:
Thinking more on this, its a problem for another PR - let it go.